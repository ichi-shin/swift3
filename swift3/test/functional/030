#!/bin/bash

# Abort Multipart Upload

. ./common

S3USER=tester

_s3_put /bucket

upload_id=$(_s3_post /bucket/object?uploads | \
    _xpath '/InitiateMultipartUploadResult/UploadId/text()')
_s3_put /bucket/object?uploadId=${upload_id}\&partNumber=1 -T /dev/null

_s3_delete /bucket/object?uploadId=${upload_id} -D - | _filter_curl

_sw_get /bucket
_sw_get /bucket+segments
