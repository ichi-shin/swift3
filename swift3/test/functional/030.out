QA output created by 030
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X POST http://SWIFT_HOST/bucket/object?uploads... 200
> s3curl --id tester -- -X PUT -T /dev/null http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID&partNumber=1... 200
> s3curl --id tester -- -X DELETE -D - http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID... 204
HTTP/1.1 204 No Content
Content-Length: LENGTH
Content-Type: text/html; charset=UTF-8
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

> curl -X GET http://SWIFT_HOST/v1/TENANT/bucket... 204
> curl -X GET http://SWIFT_HOST/v1/TENANT/bucket+segments... 204
